src/crud/useFetchMediaUrl.ts

```diff
@@ -125,21 +126,22 @@ export const useFetchMediaUrl = (reporter?: any) => {
           ) as MediaFile;
           if (mediarec && mediarec.attributes) {
             if (cancelled()) return;
+            var local = { localname: '' };
             const audioUrl =
               mediarec.attributes.audioUrl ??
               mediarec.attributes.s3file ??
               mediarec.attributes.originalFile;
-            const path = await dataPath(audioUrl, PathType.MEDIA);
-            const foundLocal = await ipc?.exists(path);
-            if (foundLocal || !accessToken) {
+            const path = await dataPath(audioUrl, PathType.MEDIA, local);
+            const foundLocal = local.localname === path;
+            if (foundLocal || offline) {
```

Comment:
line 136 says if foundlocal or offline so if offline is tru it will never get to 144... Did you mean && (and)?

Reply:
144 is inside the if of 136